### DISCUSSION TITLE
Heltec Wireless Tracker v1.2

### DISCUSSION THREAD

**Discussion by abarehh** [meshtastic/Discussions#140] [ESP32]
I just received a new Heltec Wireless Tracker. However it has v1.2 printed on the PCB. 
On meshtastic FW download and the HW support docu page only v1.1 is mentioned.
Does anybody know if v1.2 is compatible ? and may be what the differences are ?

**Comment by garthvh** [comment]
Should be fine the only difference we are aware of so far is that the spring antenna supplier changed

**Reply by abarehh** [reply]
yep, I can also confirm 2.5.15 works fine on the v1.2 board. GPS is also working fine.
However I am really disappointed with the Heltec tracker. It gets really hot, i.e. is burning energy like crazy. I knew that the ESP32 is an energy sucker and the nRFs are much more energy efficient, but this is crazy. I attached a 3000mAh LiPo and it went from 95% to 82% in like 30 min. At this rate it probably would not even last a day. Ok I have set GPS to 60sec but that cant be it. My t-echo and my t1000-e last almost 2 days with this setting (of course both are nRF based).
No wonder that the Heltec Tracker gets so warm. Its like a heater in the pocket. Not really usable for tracking

**Comment by klein-hirn** [comment]
I can confirm flashing the Heltec Wireless Tracker board v1.2 with the web-based flasher, selection Tracker V1.1, Meshtastic firmware 2.5.15 works fine.
Screen, BT and LoRa functionality confirmed. GNSS is not getting a lock in my basement for some reason ;) but is recognized and seems to be working.

**Comment by jlaughter** [comment]
It doesn't quite work perfectly.  There's a display issue, where after the display times out it won't wake up and start displaying again without a reset.  https://github.com/meshtastic/firmware/issues/6243

I think you can maybe work around it by disabling screen timeout, but that wouldn't support battery life well.

**Reply by byte-for-byte** [reply]
I confirm this issue.

**Reply by blue25i** [reply]
I also confirm this issue using 2.6.11 - the device identifies itself as "TrackSenger (big OLED)" which I dont understand as it has no big OLED screen etc. I will try to test another version / alphas...

**Reply by blue25i** [reply]
update - 2.7.5 alpha seems to work fine. Only "issue" so far is that the display activates in inverted pixels when device wakes up. But its only like some milliseconds then the display is normal.

**Reply by Jayklutch** [reply]
Thank you blue25i, I had the dreaded display not waking from sleep issue. It now also correctly displays radio settings instead of "UNSET" in top left corner.

Menu and USER_Button now also seem to work (Except caned messages)?
